dds/DCPS/InstanceState.h

```diff
@@ -230,32 +229,10 @@ class OpenDDS_Dcps_Export InstanceState : public ReactorInterceptor {
   /// the owner is updated.
   bool registered_;
 
-  struct CommandBase : Command {
-    explicit CommandBase(InstanceState* instance_state)
-      : instance_state_(instance_state)
-    {}
-
-    InstanceState* instance_state_;
-  };
-
-  struct CancelCommand : CommandBase {
-    explicit CancelCommand(InstanceState* instance_state)
-      : CommandBase(instance_state)
-    {}
-
-    void execute();
-  };
-
-  struct ScheduleCommand : CommandBase {
-    ScheduleCommand(InstanceState* instance_state, const TimeDuration& delay)
-      : CommandBase(instance_state)
-      , delay_(delay)
-    {}
-
-    const TimeDuration delay_;
-    void execute();
-  };
+  typedef PmfSporadicTask<InstanceState> InstanceStateSporadicTask;
+  RcHandle<InstanceStateSporadicTask> release_task_;
```

Comment:
I think this could be an RcHandle to the base type SporadicTask -- in which case it wouldn't need the typedef.